Tests/BoutiqueTests/BoutiqueTests.swift

```diff
@@ -2,50 +2,46 @@
 import Combine
 import XCTest
 
+@MainActor
 final class BoutiqueTests: XCTestCase {
 
     private var store: Store<BoutiqueItem>!
     private var cancellables: Set<AnyCancellable> = []
 
     override func setUp() async throws {
-        store = Store<BoutiqueItem>(storagePath: Self.testStoragePath, cacheIdentifier: \.merchantID)
+        store = await Store<BoutiqueItem>(storagePath: Self.testStoragePath, cacheIdentifier: \.merchantID)
         try await store.removeAll()
     }
 
-    @MainActor
     func testAddingItem() async throws {
         try await store.add(Self.coat)
         XCTAssert(store.items.contains(Self.coat))
     }
 
-    @MainActor
     func testAddingItems() async throws {
         try await store.add([Self.coat, Self.sweater, Self.sweater, Self.purse])
         XCTAssert(store.items.contains(Self.coat))
         XCTAssert(store.items.contains(Self.sweater))
         XCTAssert(store.items.contains(Self.purse))
     }
 
-    @MainActor
     func testAddingDuplicateItems() async throws {
         XCTAssert(store.items.isEmpty)
         try await store.add(Self.allObjects)
-        XCTAssert(store.items.count == 4)
+        XCTAssertEqual(store.items.count, 4)
     }
 
-    @MainActor
     func testReadingItems() async throws {
         try await store.add(Self.allObjects)
 
-        XCTAssert(store.items[0] == Self.coat)
-        XCTAssert(store.items[1] == Self.sweater)
-        XCTAssert(store.items[2] == Self.purse)
-        XCTAssert(store.items[3] == Self.belt)
+        XCTAssertEqual(store.items[0], Self.coat)
+        XCTAssertEqual(store.items[1], Self.sweater)
+        XCTAssertEqual(store.items[2], Self.purse)
+        XCTAssertEqual(store.items[3], Self.belt)
 
         XCTAssert(store.items.count == 4)
```

Comment:
There is no need to verify each item separately (and in the process risk crashing the test runner by accessing the array indexes without checking the count first), you can fold the whole test into:

```swift
XCTAssertEqual(store.items, [Self.coat, Self.sweater, Self.purse, Self.belt])
```

Reply:
Good idea, thanks